client/blocks/disconnect-jetpack/dialog.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * @format
+ *
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import DisconnectJetpack from '.';
+
+const DisconnectJetpackDialog = ( { isBroken, isVisible, onClose, disconnectHref, siteId } ) => (
+	<Dialog
+		isVisible={ isVisible }
+		additionalClassNames="disconnect-jetpack-dialog"
+		onClose={ onClose }
+	>
+		<DisconnectJetpack
+			disconnectHref={ disconnectHref }
+			isBroken={ isBroken }
+			onDisconnect={ onClose }
+			onStayConnected={ onClose }
+			siteId={ siteId }
+		/>
+	</Dialog>
+);
+
+DisconnectJetpackDialog.propTypes = {
+	disconnectHref: PropTypes.string,
+	isBroken: PropTypes.bool,
+	isVisible: PropTypes.bool,
+	onClose: PropTypes.func,
+	redirect: PropTypes.string,
+	siteId: PropTypes.number,
+	stayConnectedHref: PropTypes.string,
```

Comment:
It seems we don't need this one here right now. Are you planning to use it? Perhaps pass it to `<DisconnectJetpack />` eventually?

Reply:
Good question. My assumption was that in the context of a dialog, the 'Stay Connected' button would just close the dialog and nothing else, so in that vein, I could drop it. For symmetry reasons however, maybe it's better to keep (and pass) it. Won't harm I guess.